sequence/src/test/java/com/vitreoussoftware/bioinformatics/sequence/BasePairTest.java

```diff
@@ -18,130 +19,95 @@
  */
 public class BasePairTest {
 	private EncodingScheme scheme;
-	
+	private EncodingScheme otherScheme;
+
 	/**
 	 * Setup the test object
 	 */
 	@Before
 	public void setup()
 	{
-		this.scheme = new AcceptUnknownDnaEncodingScheme();
+		this.scheme = AcceptUnknownDnaEncodingScheme.instance;
+        this.otherScheme = BasicDnaEncodingScheme.instance;
 	}
 
 	/** 
 	 * Test Equality for BasePair, with same start
-	 	 */
+     */
 	@Test
 	public void testEqualityToSelf() throws InvalidDnaFormatException {
 		val bp = BasePair.create('A', scheme);
-		assertEquals(bp, bp);
+		assertThat(bp, is(bp));
 	}
-	
-	/** 
+
+
+    /**
+     * Test equality with different {@link EncodingScheme}
+     */
+    @Test
+    public void testEqualityDifferentScheme() {
+        val bp = BasePair.create('A', scheme);
```

Comment:
For this and most tests in this class, consider if this is really sufficient coverage.  Should you not perhaps be checking for each valid letter (A, T, etc.) given that the number of letters is not particularly large and behaviour (might) be different between them.